pages/users/user_reactivate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+restrict_access();
+$can_add_user = check_auth(Access::$EDIT_USERS);
+$users_repository = em()->getRepository(User::class);
+page("Réactiver des licenciés")->css("user_list.css");
+if (!empty($_POST)) {
+    foreach (array_keys($_POST) as $key) {
+        $user = $users_repository->findOneBy(['id' => $key]);
+        $user->active = true;
+        em()->persist($user);
+    }
+    em()->flush();
+}
+$users = $users_repository->findBy(['active' => "0"], ['last_name' => 'ASC', 'first_name' => 'ASC']);
+?>
+
+<form method="post">
+    <nav id="page-actions">
+        <a href="/licencies" class="secondary"><i class="fas fa-caret-left"></i> Retour</a>
+        <div>
+            <button type="submit">
+                Réactiver
+            </button>
+        </div>
+    </nav>
+
+    <input type="search" id="search-users" placeholder="Rechercher..." onkeyup="searchTable('users-table')"
+        onload="searchTable('users-table')">
```

Comment:
Same thing here as in `user_list`: I don't really understand this `onload`. I would suggest using the same logic as there was before in `user_list`